common:
    lidar_topic:  "/velodyne_points"
    odom_topic:  "/scout_base_controller/odom"
    imu_topic: "/imu/data"
    voxel_size: 0.1

gicp:
    corres_dist: 3.0
    num_thread: 4
    corres_random: 15
    iter_num: 20
    epsilon: 0.01
    euclidean_epsilon: 0.01
    ransac_iter: 5
    ransac_outlier: 1.0 

localization:
    init_x: 0.0
    init_y: 0.0
    init_z: 0.0
    init_roll: 0.0
    init_pitch: 0.0
    init_yaw: 0.0